src/stytch-provider.ts

```diff
@@ -4,12 +4,18 @@ const Pkg = require('../package.json')
 
 const Stytch = require('stytch')
 
+const https = require('https')
```

Comment:
this is system package, so place first - in general for imports follow the order:
SYSTEM
NPM
LOCAL

imported values always have a capital, thus `const Https = `